apps/react-workshop/src/ColorPicker.stories.tsx

```diff
@@ -81,9 +81,10 @@ export const Basic = () => {
           </ColorPicker>
         }
         visible={isOpen}
+        onVisibleChange={setIsOpen}
         placement='bottom-start'
       >
-        <IconButton onClick={() => setIsOpen((open) => !open)}>
+        <IconButton>
```

Comment:
Should we make this change in `ColorPicker.advancedPopover.jsx` too?

Reply:
(Missed this comment somehow)
Yes, we should fix it, but honestly, all `ColorPicker`

Reply:
(Missed this comment somehow)

Yes, we should fix it, but honestly, all `ColorPicker` examples need to be updated to use `Popover`.